use-cases/using-Intel-QAT-and-kata.md

```diff
@@ -174,7 +188,7 @@ can find more information about how to customize in the
 
 ```sh
 $ mkdir -p $QAT_CONF_LOCATION
-$ cp $QAT_SRC/build/c6xxvf_dev0.conf.vm $QAT_CONF_LOCATION/c6xxvf_dev0.conf
+$ cp $QAT_SRC/quickassist/utilities/adf_ctl/conf_files/c6xxvf_dev0.conf.vm $QAT_CONF_LOCATION/c6xxvf_dev0.conf
 $ sed -i 's/\[SSL\]/\[SHIM\]/g' $QAT_CONF_LOCATION/c6xxvf_dev0.conf
 $ for i in {1..15}; do cp $QAT_CONF_LOCATION/c6xxvf_dev0.conf $QAT_CONF_LOCATION/c6xxvf_dev$i.conf; done
```

Comment:
that `for i in...` is not necessary. it's possible to mount the same host file to multiple container files.

Reply:
I forgot to mention I removed this as well and only copy the one file.